### PR TITLE
Create BLorient4611

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record for BLorient4611 based on Strelcyns catalogue. I did not know, if I should create a LIT ID for a2 (magical prayer against enemies) and started an issue (https://github.com/BetaMasaheft/Documentation/issues/2496) just a couple of minutes ago.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient4611.xml [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+                                        <title ref="LIT2278salota"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.3">
+                                        <locus from="88r" to="89r"/>
+                                        <title ref="LIT2703salota"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.4">
+                                        <locus from="89r" to="89v"/>
+                                        <title ref="LIT2257salota"/>
+                                        <note>Indicated by the title: <q xml:lang="gez">ጸሎተ፡ ሐና፡ እመ፡ ሳሙኤል፡ ነቢይ፡</q></note>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient4611.xml:63]: possibly better `<incipit xml:lang="gez"><title>ጸሎተ፡ ሐና፡ እመ፡ ሳሙኤል፡ ነቢይ፡</title></incipit>`
https://betamasaheft.eu/Guidelines/?q=title&id=ColophSupplTit

File: LondonBritishLibrary/orient/BLorient4611.xml [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+                                        <locus from="97r" to="97v"/>
+                                        <title>Fifth section</title>
+                                        <note>Indicated by the title: <q xml:lang="gez">ኃምስ፡ መኃልይ፡ እምዚአሃ፡ ለቤተ፡ ክርስቲያን፡</q></note>
+                                    </msItem>
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="98ra" to="104vb"/>
+                                <title ref="LIT2509Weddas"/>
+                                <note>With musical notation. Arranged by the days of the week, with Monday's lesson first.</note>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient4611.xml:146]: `<notatedMusic><desc>With musical notation</desc></notatedMusic>`
`<note>Arranged by the days of the week, with Monday's lesson first.</note>`

File: LondonBritishLibrary/orient/BLorient4611.xml [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#94r"/>
+                                    <desc type="OwnershipNote">The name of the owner was
+                                        <persName role="owner" ref="PRS14357WaldaIyasusKM">Walda ʾIyasus Kǝnfa Mikāʾel</persName>.</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#Ir"/>
+                                    <desc type="AcquisitionNote"/>
+                                    <q xml:lang="en">Bought of Mr. <persName ref="PRS14356Brion">F. Brion</persName>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient4611.xml:208]: owner

File: LondonBritishLibrary/orient/BLorient4611.xml [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+                                        <date when="1893-01-26">Jan. 26. 1893</date>.</q>
+                                </item>
+                                <item xml:id="e4">
+                                    <locus target="#98ra #98va #99vb #101ra #102rb #103rb #104ra"/>
+                                    <desc type="findingAid">The text is divided into the days of the week.</desc>
+                                </item>
+                            </list>
+                        </additions>        
+                        <bindingDesc>
+                            <binding xml:id="binding">
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient4611.xml:218]: `<binding contemporary="false">`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient4611.xml [hunk 5]
```diff
@@ -35,9 +35,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <title ref="LIT2701Dawit"/>
                             <msItem xml:id="ms_i1.1">
                                 <locus from="3r" to="86v"/>
-                                <title ref="LIT2000Mazmur"/>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient4611.xml]: this should stay though

File: LondonBritishLibrary/orient/BLorient4611.xml [hunk 6]
```diff
@@ -59,13 +58,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </msItem>
                                     <msItem xml:id="ms_i1.2.1.4">
                                         <locus from="89r" to="89v"/>
-                                        <title ref="LIT2257salota"/>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient4611.xml]: also here, etc. put all titles back and reread the guidelines link i had provided, thank you

File: LondonBritishLibrary/orient/BLorient4611.xml (hunk 5 above)
Comment (eu-genia) [reply LondonBritishLibrary/orient/BLorient4611.xml]: markup of the `incipit `is not the same as `msItem`, these are two different things, `msItem `needs a `title`, `incipit `may have a `type `or an inner markup

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by thea-m — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**